### PR TITLE
Fix golint warnings in cgroup directory

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
Let us enable gometalinter in the build so we fix everything at once and it stays fixed. WDYT?

**Comment by rhatdan** [comment]
Makes sense to me.

**Comment by hqhq** [comment]
@mrunalp , I was planning to do so, but there's too many golint warning, we can find it from [Go Report Card](https://goreportcard.com/report/github.com/opencontainers/runc#golint) now, cgroup is the biggest part from I can see. I think we can fix all of them before adding it to CI.

The change would touch too much files, do it in once would likely cause conflicts and needs many rebases, so I think it makes sense to separate it to a few PRs, I'll make the rest PRs if no one else do it in the next few days.

**Comment by mrunalp** [comment]
@hqhq sounds good. I'll fix a few of them.

File: libcontainer/cgroups/systemd/apply_systemd.go [hunk 1]
```diff
@@ -468,6 +476,7 @@ func (m *Manager) Freeze(state configs.FreezerState) error {
 	return nil
 }
 
+// GetPids returns all pids, that were added to cgroup.
```
Comment (mrunalp) [inline libcontainer/cgroups/systemd/apply_systemd.go]: nit: don't need the comma: .. all pids that were ..

File: libcontainer/cgroups/systemd/apply_systemd.go [hunk 2]
```diff
@@ -476,6 +485,8 @@ func (m *Manager) GetPids() ([]int, error) {
 	return cgroups.GetPids(path)
 }
 
+// GetAllPids returns all pids, that were added to cgroup and
```
Comment (mrunalp) [inline libcontainer/cgroups/systemd/apply_systemd.go]: Same here.

**Comment by hqhq** [comment]
@mrunalp Updated.

**Comment by hqhq** [comment]
Rebased.
ping @mrunalp @cyphar @crosbymichael @dqminh Can you do a quick review and merge this?

Also please pay attention to https://github.com/opencontainers/runc/pull/1237 , CI failed frequently (including this PR) and that PR will fix it.

**Comment by dqminh** [comment]
@hqhq can you rebase one more time ? :pray:

**Comment by AkihiroSuda** [comment]
Closing because this one is quote outdated, but feel free to reopen.

**Pull request closed without merging**